src/config.ml

```diff
@@ -36,6 +36,29 @@ open Stanza.Decoder
    files for simplicity *)
 let syntax = Stanza.syntax
 
+module Build_logs = struct
+  type t =
+    | Preserve
+    | Clear_on_rebuild
+
+  let error =
```

Comment:
I would inline this in `of_string`. We don't usually extract error messages unless they are big or shared between several functions.

Reply:
Done